Controller/CRUDController.php

```diff
@@ -465,7 +465,17 @@ public function batchAction()
         if (count($idx) > 0) {
             $this->admin->getModelManager()->addIdentifiersToQuery($this->admin->getClass(), $query, $idx);
         } elseif (!$allElements) {
-            $query = null;
+            $this->addFlash(
+                'sonata_flash_info',
```

Comment:
I've didn't found where this flash type `sonata_flash_info` is set. On my current project, the flash message is shown as a warning message instead of an info message. According to this documentation https://sonata-project.org/bundles/core/master/doc/reference/flash_messages.html , only success, warning and error are defined as default messages types.

Reply:
Please use `warning` then.

Reply:
Just found out `sonata_flash_info` is well defined here:

https://github.com/sonata-project/SonataCoreBundle/blob/87ed618efe4b3c51694ab7d057656912850f6dc7/DependencyInjection/SonataCoreExtension.php#L150-L153

Please use it.